client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -35,7 +35,7 @@ struct SendError {
   };
   ErrorType type;
 };
-typedef std::variant<SendError, bft::client::Reply> SendResult;
+typedef std::variant<concord_client_pool::SubmitResult, bft::client::Reply> SendResult;
```

Comment:
At some point, concord_client.hpp should be self-contained so that we don't make any references to the client pool  or the TRC. With this change, client pool knowledge is required inside the request service. We implicitly included TRC and client_pool in the clientservice already. Nothing to change here, just FYI :)

Reply:
Agree I just didn't want to expose unrelated things in the pool right now.